policyengine_uk/tests/policy/integration/income.yaml

```diff
@@ -0,0 +1,34 @@
+- name: Earned income is taxed correctly
+  period: 2020
+  absolute_error_margin: 0
+  input:
+    employment_income: 100000
+  output:
+    allowances: 12500
```

Comment:
```suggestion
    allowances: 12_500
```

Reply:
In general we use "_" as "," for clarity with large numbers (each set threes)

Reply:
I'm using too much Claude haha

Reply:
no such thing as too much